accounts.rb

```diff
@@ -0,0 +1,36 @@
+require_relative 'bank.rb'
+class Accounts
+  include Bank
+
+  def initialize
+    account_info
+  end
+
+  def acc_info
+    acc = {
+      name: @page.css('span.user-name').text,
+      balance: @page.css('span.amount').first.text,
+      currency: @page.css('span.amount.currency.MDL').text,
+      nature: 'visa'
+    }
+  end
+
+  def saving_acc_info
+    acc = { 'accounts' => [] }
```

Comment:
strange that Your Account class has this logic. The task asks you to create an array of accounts, it means you should have an array somewhere that contains 1 or more objects of Account class.